### DISCUSSION TITLE
Using pyenv version of python instead of installing another version in homebrew

### DISCUSSION THREAD

**Discussion by tcarroll2** [Homebrew/discussions#616] [Everyday usage]
Is there a way I can tell brew that there is already a python version installed via pyenv?  I don't see why I need multiple copies of python taking up disk space on my system.

**Comment by SMillerDev** [comment]
There is not. Homebrew only tests it's dependencies against homebrew software so we can't guarantee that it'll work if you use a pyenv Python
